src/blueprints/Do.Blueprints.Service.Application/Orm/Default/CorruptedJsonDataExceptionHandler.cs

```diff
@@ -13,6 +13,6 @@ public class CorruptedJsonDataExceptionHandler : IExceptionHandler
     public ExceptionInfo Handle(Exception ex) =>
         new(
             (int)HttpStatusCode.InternalServerError,
-            ex.InnerException?.Message ?? ex.Message
+            "An unexpected error occurred."
```

Comment:
remove this handler, since we are not sharing the exception message of an internal server error any more